.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  test:
```

Comment:
should we call it `linkChecker` instead of `test` like in the linkchecker repo's example https://github.com/peter-evans/link-checker?